src/generic/generic-component-message/data/demo--info.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   dismiss: true,
-  srOnly: 'Informative message',
```

Comment:
I was expecting that you to rename it `screen_reader_description` instead of `srOnly`, not to delete it...